test/functional/wallet_resendwallettransactions.py

```diff
@@ -64,7 +64,7 @@ def run_test(self):
         # Transaction should be rebroadcast approximately 24 hours in the future,
         # but can range from 12-36. So bump 36 hours to be sure.
         node.setmocktime(now + 36 * 60 * 60)
-        node.p2p.wait_for_broadcast([txid])
+        peer_second.wait_for_broadcast([txid])
```

Comment:
6 lines above there's a stray `node.p2ps[1]` that can be replaced with `peer_second`.

Reply:
@guggero Seems like a good suggestion. Mind following up with a patch?